relentless/potential/potential.py

```diff
@@ -3,6 +3,7 @@
 import abc
 import json
 import numpy as np
+import warnings
```

Comment:
PEP8 (https://www.python.org/dev/peps/pep-0008/#imports) recommends that you import in the following order:
1. Standard library
2. Third-party
3. Local
with a blank line between each of them.

So here:
```suggestion
import warnings

import numpy as np
```